io.github.heathcliff26.go-minesweeper.yaml

```diff
@@ -0,0 +1,59 @@
+app-id: io.github.heathcliff26.go-minesweeper
+runtime: org.freedesktop.Platform
+runtime-version: '24.08'
+sdk: org.freedesktop.Sdk
+command: go-minesweeper
+finish-args:
+  - --socket=x11
+  - --share=ipc
+  - --device=dri
+modules:
+  - name: go-minesweeper
+    buildsystem: simple
+    sources:
+      - type: file
+        dest-filename: go-minesweeper.tar.gz
+        only-arches: [x86_64]
+        url: https://github.com/heathcliff26/go-minesweeper/releases/download/v0.5.5/go-minesweeper_linux-amd64.tar.gz
+        sha256: 3f166e831fd0e48adc3577617e1566211722ce79b795396428f45404af305cf8
+        x-checker-data:
+          is-main-source: true
+          type: json
+          url: https://api.github.com/repos/heathcliff26/go-minesweeper/releases/latest
+          version-query: .tag_name
+          timestamp-query: .published_at
+          url-query: .assets | map(.browser_download_url | select(endswith("_linux-amd64.tar.gz"))) | first
+      - type: file
+        dest-filename: go-minesweeper.tar.gz
+        only-arches: [aarch64]
+        url: https://github.com/heathcliff26/go-minesweeper/releases/download/v0.5.5/go-minesweeper_linux-arm64.tar.gz
+        sha256: 2c4d3eef5dd8597947b469ec65eec2ed66aad635bac1cef61c8194cfd2ff6264
+        x-checker-data:
+          is-main-source: true
+          type: json
+          url: https://api.github.com/repos/heathcliff26/go-minesweeper/releases/latest
+          version-query: .tag_name
+          timestamp-query: .published_at
+          url-query: .assets | map(.browser_download_url | select(endswith("_linux-arm64.tar.gz"))) | first
+      - type: file
+        dest-filename: go-minesweeper.png
+        url: https://raw.githubusercontent.com/heathcliff26/go-minesweeper/refs/tags/v0.5.5/img/mine.png
+        sha256: 31e564f9becc333cd03910d1f2530b5653b182d50e93d9f41229cfb4ada16cd6
+        x-checker-data:
+          is-main-source: true
+          type: json
+          url: https://api.github.com/repos/heathcliff26/go-minesweeper/releases/latest
+          version-query: .tag_name
+          timestamp-query: .published_at
+          url-query: '"https://raw.githubusercontent.com/heathcliff26/go-minesweeper/refs/tags/" + $tag + "/img/mine.png"'
+      - type: file
+        path: io.github.heathcliff26.go-minesweeper.desktop
+      - type: file
+        path: io.github.heathcliff26.go-minesweeper.metainfo.xml
+
+    build-commands:
+      - tar -xzf go-minesweeper.tar.gz
```

Comment:
source should be of type: `archive` instead

Reply:
done